csunplugged/topics/management/commands/loadtopics.py

```diff
@@ -21,12 +21,14 @@ def handle(self, *args, **options):
                 attribute.
         """
         factory = LoaderFactory()
+
         # Get structure and content files
         base_loader = BaseLoader()
-        BASE_PATH = "topics/content/en/"
+        base_path = "topics/content/"
```

Comment:
Could this be moved to the settings file? I presume this would then make it easier to test `loadtopics` which isn't currently tested.

Reply:
Agreed, however I suggest this is done at the same time as writing tests for `loadtopics.py`, which should be in a separate PR after #677 is merged (or as part of that PR).